### DISCUSSION TITLE
How to use BlazorXtabs with Authorization?

### DISCUSSION THREAD

**Discussion by efabian1** [David-Moreira/BlazorXTabs#31] [Q&A] [answered]
XTabsRouteView  is a router but I need 
      something like AuthorizeXTabsRouteView  or a way to subtitute AuthorizeRouteView

**Comment by David-Moreira** [comment]
Hello @efabian1 
You're right. I'll take a look at Authorization. Most likely expect an `AuthorizeXTabsRouteView`

**Comment by David-Moreira** [comment, marked as answer]
@efabian1 Please find the new `XTabsAuthorizeRouteView` in the newest version, **1.6.0**
https://www.nuget.org/packages/BlazorXTabs/1.6.0
